seisidd/experiment.py

```diff
@@ -108,8 +120,83 @@ def calibration(self):
         # TODO:
         #  Still not clear how calibration can be done automatically, but
         #  let's keep a function here as a place holder
+        #  Check out this auto alignment to see if some functions can be used here
+        #  https://github.com/AdvancedPhotonSource/auto_sample_alignment.git
+        #  Per conversation with Peter, This package can return the same location on the pin
+        #  according to the images.  However, they are requesting more features like determine 
+        #  the slit position and size.
+        #  Jun and Peter will test this code during the first week of October, let wait for their feedback.
+
+
         pass
-        
+
+    @staticmethod
+    def get_tomobeam(mode):
+        """return Tomobeam based on given mode"""
+        if mode.lower() in ['dryrun', 'production']:
+            beam = Beam()
+        elif mode.lower() == 'debug':
+            # NOTE:
+            #   This is a place holder for maybe additional control of the beam
+            #   in the debug mode.  Let's discuss what should be exposed here.
+            #   Or, we just use some simulated motors for the beam here as below?
+            from ophyd import sim
```

Comment:
The current simulated devices from Ophyd is not very useful for us.  We are better off using simulated devices from Epics.